PyPDF2/generic/_base.py

```diff
@@ -94,7 +110,9 @@ def clone(
         """
         raise Exception("clone PdfObject")
 
-    def _reference_clone(self, clone: Any, pdf_dest: "PdfWriter") -> "PdfObject":
+    def _reference_clone(
+        self, clone: Any, pdf_dest: _PdfWriterInterface
```

Comment:
Here the interface is fine as it's only PyPDF2 internal

Reply:
reviewed to ease readability